README.dev.md

```diff
@@ -380,7 +380,7 @@ Then you should create the Docker image based on Ubuntu Linux, first
 go to the directory with the Zulip source code:
 
 ```
-docker build -t user/zulipdev .
+docker build -f Dockerfile-dev -t user/zulipdev .
```

Comment:
The `-f Dockerfile-dev` is only me needed for the `docker build` and not for the `docker run` command.